sdk/dnsresolver/Azure.ResourceManager.DnsResolver/src/Generated/DnsSecurityRuleData.cs

```diff
@@ -57,16 +56,15 @@ public partial class DnsSecurityRuleData : TrackedResourceData
         /// <param name="location"> The location. </param>
         /// <param name="priority"> The priority of the DNS security rule. </param>
         /// <param name="action"> The action to take on DNS requests that match the DNS security rule. </param>
-        /// <param name="dnsResolverDomainLists"> DNS resolver policy domains lists that the DNS security rule applies to. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="action"/> or <paramref name="dnsResolverDomainLists"/> is null. </exception>
-        public DnsSecurityRuleData(AzureLocation location, int priority, DnsSecurityRuleAction action, IEnumerable<WritableSubResource> dnsResolverDomainLists) : base(location)
```

Comment:
We need to mitigate this breaking change in this way:
Create a partial class of this class in `src/Custom` with the same filename as this with the following content:
```C#
namespace Azure.ResourceManager.DnsResolver
{
	public partial class DnsSecurityRuleData 
	{
		public DnsSecurityRuleData(AzureLocation location, int priority, DnsSecurityRuleAction action, IEnumerable<WritableSubResource> dnsResolverDomainLists) : this(location, priority, action)
		{
			DnsResolverDomainLists = dnsResolverDomainLists.ToList();
		}
	}
}
```